crates/bevy_reflect/bevy_reflect_derive/src/container_attributes.rs

```diff
@@ -76,6 +77,10 @@ impl TraitImpl {
 #[derive(Clone, Default)]
 pub(crate) struct FromReflectAttrs {
     auto_derive: Option<LitBool>,
+    /// Exclusively for creating default values in [`impl_reflect`]'s `FromReflect` impl.
```

Comment:
since the PR mandates that all declared fields must be specified, the attribute only does anything when the crate invoking the macro doesn't own the type its invoked on. this is only relevant for `impl_reflect!` currently, but will be needed for remote types.